packages/paste-core/primitives/combobox/stories/index.stories.tsx

```diff
@@ -243,7 +265,7 @@ export const BasicMultiCombobox: React.FC = () => {
         )}
       </Box>
       <FormPillGroup {...formPillState} aria-label="Selected components">
-        {selectedItems.map((item, index) => {
+        {selectedItems.map((selectedItem, index) => {
```

Comment:
Updated to call the iterated item `selectedItem` because it matches how dropshift recommends doing it: https://downshift.netlify.app/use-multiple-selection#usage-with-combobox